### PR TITLE
Use python binary to run multiprocess tests.

### PR DESCRIPTION
**Pull request by jdm**
The python2 binary doesn't exist in the path on macOS. This change uses an existing python2 on macOS and whatever the default python version is on linux.

### CODE REVIEW

**Comment by SimonSapin** [comment]
LGTM, but if this `python2` fails now how did it work before?

**Comment by jdm** [comment]
It never worked. I think possibly the `| cat` on the end of the line suppresses the error so CI doesn't report a failure.

**Comment by SimonSapin** [comment]
Please also add `-o pipefail` (before `-c`) at these two locations, in order to catch such failures:
https://github.com/servo/servo/blob/5f5643e7aa1b532917794cb739f8973539b45fc8/etc/taskcluster/decisionlib.py#L752
https://github.com/servo/servo/blob/5f5643e7aa1b532917794cb739f8973539b45fc8/etc/taskcluster/decisionlib.py#L688

File: etc/taskcluster/decision_task.py [hunk 1]
```diff
@@ -614,7 +614,7 @@ def wpt_chunks(platform, make_chunk_task, build_task, total_chunks, processes,
         # https://github.com/servo/servo/issues/22438
         if this_chunk == 0:
             task.with_script("""
-                time python2 ./mach test-wpt --release --binary-arg=--multiprocess \
+                time python ./mach test-wpt --release --binary-arg=--multiprocess \
```
Comment (SimonSapin) [inline etc/taskcluster/decision_task.py:617]: `time ./mach` (relying on shebang, like in other uses of `mach` commands) should also work, but this is fine.

**Review by SimonSapin — APPROVED** [review]
r+ assuming try looks good

**Comment by jdm** [comment]
@bors-servo retry
* #26233 #25513

**Comment by jdm** [comment]
@bors-servo retry
* #25258

**Pull request merged**